src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8572,11 +8572,37 @@ def wrapped_func(*args, **kwargs):  # type: ignore[no-untyped-def] # pragma: no
     def _apply_snowpark_python_function_to_columns(
         self,
         snowpark_function: Callable,
+        kwargs: Any,  # possible named arguments which need to be added
```

Comment:
should the typing here be `dict[Any, Any]`? also, should we make it `**kwargs` to match the usual python convention?

Reply:
I typed this as dict[str, Any] ( which is sort of similar to how we type other internal functions ). I can't do **kwargs here because it's an explicit argument passed through from other functions as opposed to an optional set of arguments.